language/move-stdlib/src/natives/string.rs

```diff
@@ -24,8 +23,21 @@ use std::collections::VecDeque;
 // create a `&str` view on the bytes without a copy. Once we have this
 // view, we can call ut8 functions like length, substring, etc.
 
-pub fn native_check_utf8(
-    context: &mut NativeContext,
+/***************************************************************************************************
```

Comment:
I appreciate the comments :) But they could maybe be a tad smaller ;) Not needed to change though if you like it. 

I just feel the size maybe outweighs the importance. A `///` might be fine enough